### PR TITLE
Update non-responsive header css to include new mega menu styles

### PR DESCRIPTION
**Pull request by virginiacc**
Adds new mega menu header/footer styles to nonresponsive header css

## Additions
- Non-responsive css updates

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css [hunk 1]
```diff
@@ -1113,6 +1124,10 @@ button#skip-nav:focus::-moz-focus-inner,input#skip-nav:focus::-moz-focus-inner {
     margin-left: .33333333em;
 }
 
+.o-mega-menu {
+        color: red !important;
```
Comment (anselmbradford) [inline cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css]: TRIVIAL: The indentation is too much here.

File: cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css [hunk 2]
```diff
@@ -1052,6 +1052,17 @@ p+.m-global-banner_head,ul+.m-global-banner_head,ol+.m-global-banner_head,dl+.m-
...
 }
 
+.o-mega-menu .nav_footer {
+    border-top: 1px solid #b4b5b6;
+    padding: 15px 0
+    margin-top: 30px;
+    font-size: 1.125em;
+}
+.o-mega-menu_content-2-overview{
+  margin-bottom:20px;
```
Comment (anselmbradford) [inline cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css]: TRIVIAL: The indentation is too little here. Also needs a space after the `:`.

File: cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css [hunk 3]
```diff
@@ -1052,6 +1052,17 @@ p+.m-global-banner_head,ul+.m-global-banner_head,ol+.m-global-banner_head,dl+.m-
     display: block;
 }
 
+.o-mega-menu .nav_footer {
+    border-top: 1px solid #b4b5b6;
+    padding: 15px 0
```
Comment (anselmbradford) [inline cfgov/legacy/static/nemo/_/c/v1-nonresponsive-header.css]: Missing an end of line semicolon.

**Comment by virginiacc** [comment]
@anselmbradford All the layout issues should be fixed -- thanks for catching them!

**Pull request merged**